gno/r/polls/json.gno

```diff
@@ -0,0 +1,15 @@
+package polls
+
+import (
+	"gno.land/p/demo/json"
+	pollsv1 "gno.land/p/zenao/polls/v1"
+)
+
+func PollToJSON(poll *pollsv1.Poll) string {
```

Comment:
```suggestion
func pollToJSON(poll *pollsv1.Poll) string {
```

Reply:
why ?
should i edit this too: https://github.com/samouraiworld/zenao/blob/main/gno/r/social_feed/json.gno
people can still use function unexported ?

Reply:
in queries yes and there is no upside in exposing this, the downside is that it will appear in the gnoweb help and docs